src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -256,4 +268,7 @@
                     az vm capture -g my_rg -n my_vm_name --vhd-name-prefix my_prefix\n\r
                     """
 
-
+helps[`vm wait`] = """
+    type: command
+    short-summary: Place the VM in a waiting state until a condition is met.
```

Comment:
Here it is the CLI, not the VM, that's put into a waiting state based on the VM meeting some condition.